llvm/include/llvm/CodeGen/ValueTypes.h

```diff
@@ -185,7 +185,8 @@ namespace llvm {
 
     /// Return true if the type is a scalable type.
     bool isScalableVT() const {
-      return isScalableVector() || isScalableTargetExtVT();
+      return isScalableVector() || isScalableTargetExtVT() ||
+             isRISCVVectorTuple();
```

Comment:
Should isRISCVVectorTuple() be part of isScalableTargetExtVT()?

Reply:
Yeah, it should